src/server/pkg/storage/chunk/writer.go

```diff
@@ -15,83 +15,73 @@ import (
 const (
 	// MB is Megabytes.
 	MB = 1024 * 1024
-	// AverageBits determines the average chunk size (2^AverageBits).
-	AverageBits = 23
 	// WindowSize is the size of the rolling hash window.
 	WindowSize = 64
 )
 
 var initialWindow = make([]byte, WindowSize)
```

Comment:
If I'm understanding the use of this correctly, this is 64 byte padding we apply at the beginning to prevent bytes in the previous chunk affecting the chunking algorithm. If so I think this should be renamed to `padding` or something like that. The word "window" really doesn't communicate its usage to me.